pkg/kubelet/prober/worker.go

```diff
@@ -178,7 +178,13 @@ probeLoop:
 		case <-w.stopCh:
 			break probeLoop
 		case <-probeTicker.C:
+			klog.V(4).InfoS("Triggerd Probe by periodSeconds", "probeType", w.probeType, "pod", klog.KObj(w.pod), "podUID", w.pod.UID, "containerName", w.container.Name)
```

Comment:
I don't think this message is necessary because every probe emits a level 4 log when it runs:
https://github.com/kubernetes/kubernetes/blob/de6db3ffb0680f2022f8795038eb2ca5b36c5bd6/pkg/kubelet/prober/prober.go#L136-L179
So, we can distinguish whether a probe is periodical or manual by searching the message for manual run added below.

In addition, this message looks noisy because this is emitted even while the probe in on hold. It means this message is logged for a startup probe while the container is running.

Reply:
@hshiina 
Indeed, you are right.
I've removed the periodical log.

Reply:
/lgtm